eclass/gkrellm-plugin.eclass

```diff
@@ -96,6 +102,5 @@ gkrellm-plugin_src_install() {
 		fi
 	fi
 }
-
 _GKRELLM_PLUGIN_R1=1
```

Comment:
move this directly after `if [[ ! ${_GKRELLM_PLUGIN_R1} ]]; then`

Reply:
Done.